Ships as Raw.txt

```diff
@@ -637,7 +656,7 @@ RearArcConjunction|62
 
 Name|Imperial Star Destroyer Kuat Refit

```

Comment:
This plus line 686, looks suspiciously like a repeat.  Am I correct, or did we just miss renaming this entry?